### ISSUE TITLE
historical data not completely retrieved

### ISSUE THREAD

**Issue by hylken** [alvarob96/investpy#21]
The following yields a dataframe with only 137 days of data:
df = investpy.get_etf_historical_data(etf='Vanguard FTSE Emerging Markets UCITS USD Inc', from_date='01/01/2000', to_date='12/08/2019', as_json=False, order='ascending', debug=False)

However, there should be multiple years of data (2013-present):
https://www.investing.com/etfs/vanguard-ftse---emerging-markets-historical-data?cid=962110

**Comment by alvarob96** [comment]
Hi @hylken, 

The issue you are describing is due to __"multiple ETFs with the same name and/or symbol operating in different countries"__, so an additional parameter is needed when it comes to ETF retrieval.

This parameter is the `country` of the ETF, so there are no errors related to either duplicated ETF names or symbols. So on, the functions will be fixed as:

```python
investpy.get_etf_historical_data(etf='Vanguard FTSE Emerging Markets UCITS USD Inc',
                                 country='netherlands',
                                 from_date='01/01/2000',
                                 to_date='12/08/2019',
                                 as_json=False,
                                 order='ascending',
                                 debug=False)
```

With this fix, the function will return the historical data of the introduced ETF from the selected country. This fix will be updated on 0.8.9 release as soon as possible, so [watch the repository](https://github.com/alvarob96/investpy) to get notified whenever it is fixed and updated.

Thank you for your feedback and help! 🚀

**Comment by alvarob96** [comment]
The issue has been solved and tested in commit 15f0738e77e16bc1603d82c9b64cecec189eb150, since tests failed due to missing `country` parameter.

:white_check_mark: Build is now __passing__ on Travis-CI, Azure Pipelines and Github Actions. 👍

**Comment by hylken** [comment]
Thank you!

Op ma 2 sep. 2019 om 07:13 schreef Álvaro Bartolomé del Canto <
notifications@github.com>:
